README.md

```diff
@@ -30,12 +30,12 @@ To test the starting-kit, run
 
 
 ```
-ramp_test_submission --quick-test
+ramp-test --quick-test
 ```
 
 
 #### Help
-Go to the `ramp-workflow` [wiki](https://github.com/paris-saclay-cds/ramp-workflow/wiki) for more help on the [RAMP](http:www.ramp.studio) ecosystem.
+Go to the `ramp-workflow` [wiki](https://github.com/paris-saclay-cds/ramp-workflow/wiki) for more help on the [RAMP](https:ramp.studio) ecosystem.
```

Comment:
I guess it should be `https://www.ramp.studio`

Reply:
I did change it directly editing README on github